src/components/Stake/Stake.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Header } from '@aragon/ui'
+import LayoutBox from '../Layout/LayoutBox'
+import LayoutColumns from '../Layout/LayoutColumns'
+import LayoutGutter from '../Layout/LayoutGutter'
+import LayoutLimiter from '../Layout/LayoutLimiter'
+import SideBar from './SideBar'
+
+const Stake = React.memo(function Stake() {
+  return (
+    <LayoutGutter
+      css={`
+        position: relative;
+        z-index: 2;
```

Comment:
Just checking whether we need these values here, z-index should already be being handled higher in the tree.

Reply:
good catch! this wasnt meant to be here